rir/src/runtime/RirRuntimeObject.h

```diff
@@ -74,6 +74,7 @@ struct RirRuntimeObject {
     RirRuntimeObject(uint32_t gc_area_start, uint32_t gc_area_length)
         : info{gc_area_start, gc_area_length, MAGIC} {
         uint8_t* start = (uint8_t*)this + gc_area_start;
+        // cppcheck-suppress pointerSize
```

Comment:
Yeah, this is indeed weird

Reply:
The commit where it breaks is this one https://github.com/reactorlabs/rir/pull/1282/commits/81f05b91284ceecb467673cb3e97f566bd135497, which only moves code around, so it is weird that CppChecks triggers only after that.

Reply:
The same error is raised even on the newest CppCheck version (2.14.1).